editor.planx.uk/src/@planx/components/Pay/Editor/Editor.tsx

```diff
@@ -69,9 +69,9 @@ const Component: React.FC<Props> = (props: Props) => {
                 <Input
                   format="data"
                   name="fn"
-                  value={values.fn}
+                  value={PASSPORT_FN}
                   placeholder="Data field for calculated amount"
```

Comment:
nit: can we also remove this confusing placeholder (eg "calculated" not "payable" language) since will never be seen now?

Reply:
Good catch! Removed 🗑️